proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -118,8 +118,11 @@ message Validator {
     (gogoproto.nullable)   = false
   ];
 
-  // Strictly positive if this validator's unbonding has been stopped by external modules
+  // strictly positive if this validator's unbonding has been stopped by external modules
   int64 unbonding_on_hold_ref_count = 12;
+
+  // list of unbonding ids, each uniquely identifing an unbonding of this validator
+  repeated uint64 unbonding_ids = 13;
```

Comment:
This is only ever used for deleting index entries, right?

Reply:
Unfortunately, yes. I'm not thrilled with the solution, but I cannot think of another one. :( 

Usually though it will be either empty or with only one unbonding ID.